### PR TITLE
[WIP][0.17] Port new recordevents

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -55,4 +55,5 @@ replace (
 	k8s.io/apiserver => k8s.io/apiserver v0.17.6
 	k8s.io/client-go => k8s.io/client-go v0.17.6
 	k8s.io/code-generator => k8s.io/code-generator v0.17.6
+	knative.dev/eventing => /home/slinkydeveloper/go/src/knative.dev/eventing
```
Comment (matzew) [inline go.mod:58]: @slinkydeveloper   that path?

Comment (slinkydeveloper) [reply go.mod:58]: That's why it's wip :smile:

Comment (slinkydeveloper) [reply go.mod:58]: https://github.com/openshift/knative-eventing/pull/898#issuecomment-712699954

**Pull request closed without merging**